assets/components/adFreeSection/AdFreeSection.jsx

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+// ----- Imports ----- //
+import React from 'react';
+import GridImage from 'components/gridImage/gridImage';
+import LeftMarginSection from 'components/leftMarginSection/leftMarginSection';
+
+export default function AdFreeSection() {
+
+  return (
+    <LeftMarginSection modifierClasses={['blue']}>
+      <div className="component-ad-free">
```

Comment:
By convention we make the `className` match the component name, so in this case it would become `component-ad-free-section`. We also try to scope everything in the component within the [BEM "block"](http://getbem.com/naming/), so here `LeftMarginSection` would live inside `<div className="component-ad-free-section">`.